datasets/office31.py

```diff
@@ -25,33 +33,45 @@ class Dataset(BaseDataset):
             ('amazon', 'dslr'),
             ('amazon', 'webcam')
         ],
-        'n_components': [100]
     }
 
+    path_dataset = "data/OFFICE31.zip"
+    path_extract = "data/OFFICE31/"
+    url_dataset = "https://wjdcloud.blob.core.windows.net/dataset/OFFICE31.zip"
+
+
+    def _get_dataset(self, domain_select):
+        # Define transformations to preprocess the images
+        preprocess = transforms.Compose([
+            transforms.Resize(256),
+            transforms.CenterCrop(224),
+            transforms.ToTensor(),
+            transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
```

Comment:
where did you get these values?

Reply:
imagenet normalisation terms since we'll be using a pertained resnet50